Makefile

```
@@ -1,7 +1,7 @@
 CHANNELS  = $(addprefix -c ,$(shell tr '\n' ' ' <$(RECIPE_DIR)/channels)) -c local
 METADEPS  = $(addprefix $(RECIPE_DIR)/,meta.yaml) src/uwtools/resources/info.json
 METAJSON  = $(RECIPE_DIR)/meta.json
-TARGETS   = clean-devenv devshell docs env format lint meta package test typecheck unittest
+TARGETS   = clean-devenv devshell docs env format lint meta package test typecheck unittest test_notebooks
```

```suggestion
TARGETS   = clean-devenv devshell docs env format lint meta package test test-notebooks typecheck unittest
```
To keep the lexicographical ordering of the target names, and for consistency with hyphenated `clean-devenv`, and the general convention that command-line arguments are hyphenated and seldom use underscores.

I do see some Python projects using underscores on the command line because it's more convenient given that internal Python identifiers have to use underscores, but Unix tradition has preferred hyphens. For example:

```
$ find /bin /usr/bin /usr/local/bin -type f | grep '-' | wc -l
495
```

vs

```
$ find /bin /usr/bin /usr/local/bin -type f | grep '_' | wc -l
105
```

Those are just program names, but you get the idea. I'm kind of disappointed to see that many underscores -- not as "seldom" as I'd hoped, though a lot of the programs appear to be from one or two closely-related families of tools. I don't see the benefit to making (what looks to me like) an arbitrary change that will keep users guessing.

---

Ah, thanks! That's a Unix tradition that I wasn't aware of. Fixed this in the latest commit.